### PR TITLE
Initial draft of the GMake user manual.

### PR DESCRIPTION
**Pull request by rsbeckerca**
Fixes: #47

CLA: Permission granted to the ITUGLIB Engineering Team.

Signed-off-by: Randall S. Becker <rsbecker@nexbridge.com>

### CODE REVIEW

**Comment by rsbeckerca** [comment]
This is a first draft of the documentation. It was requested by an ITUGLIB user, so I threw something together given the current state of affairs in GMake. It does not include upcoming features.

File: MANUAL.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+### 7. AGGREGATION WITH INDEPENDENT WORKS
+
+A compilation of the Document or its derivatives with other separate and independent documents or works, in or on a volume of a storage or distribution medium, is called an �aggregate� if the copyright resulting from the compilation is not used to limit the legal rights of the compilation�s users beyond what the individual works permit. When the Document is included in an aggregate, this License does not apply to the other works in the aggregate which are not themselves derivative works of the Document.
+
+If the Cover Text requirement of section 3 is applicable to these copies of the Document, then if the Document is less than one half of the entire aggregate, the Document�s Cover Texts may be placed on covers that bracket the Document within the aggregate, or the electronic equivalent of covers if the Document is in electronic form. Otherwise they must appear on printed covers that bracket the whole aggregate.
+### 8. TRANSLATION
```
Comment (Jojo-Schmitz) [inline MANUAL.md:251]: line feed missing?

File: MANUAL.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+## Extensions
+
+The following extensions are currently defined for GMake:
+
+* GUARDIAN DEFINE variable expansion.
+* Program execution using `PROCESS_LAUNCH_` instead of shell.
+* Recognition of disk and process names instead of single character variables.  
+
+## Appendix A
+Appendix D GNU Free Documentation License
```
Comment (Jojo-Schmitz) [inline MANUAL.md:148]: linefeed missing?

File: MANUAL.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+run through an `IN` file. The following run options are currently supported and
+are all optional:
+
+| Option | Argument | Meaning |
+| ------ | -------- | ------- |
+|   `IN`   | file     | The file to be used as an input file to the program. This is generally used. |
+|  `OUT`   | file or spooler | The file or location to be used to capture output. This is optional. |
+|  `CPU`   | number   | The CPU on which the program will be run. |
+| `NAME`   | pname    | The name of the process to be run. |
+| `DEBUG`  |          | Causes the program to be started in DEBUG mode. |
```
Comment (Jojo-Schmitz) [inline MANUAL.md:128]: Table looks good as MarkDown, but might get improved for plain text

**Review by Jojo-Schmitz — APPROVED** [review]

**Review by Jojo-Schmitz — APPROVED** [review]

**Pull request merged**